### PR TITLE
Trigger login from App

### PR DESCRIPTION
**Pull request by carabj**

### CODE REVIEW

File: example-backend/src/main/kotlin/app/truid/example/examplebackend/ExampleBackendApplication.kt [hunk 1]
```diff
@@ -30,7 +33,12 @@ class ExampleBackendApplication {
 
     @Bean
     fun webClient(): WebClient {
-        return WebClient.create()
+        return WebClient
+            .builder()
+            // create a new connection for each request to avoid connection reset by peer when docker compose container
+            // is restarted
```
Comment (johanblumenberg) [inline example-backend/src/main/kotlin/app/truid/example/examplebackend/ExampleBackendApplication.kt:39]: Docker compose container is truid backend? The audience for the example code doesn't have one of those and will not understand what this comment is.

Is there a connection pool that doesn't realize when a connection is reset? Doesn't the docker containers close all sockets when terminating?

Maybe easier to configure the client not to use a connection pool?

Comment (carabj) [reply example-backend/src/main/kotlin/app/truid/example/examplebackend/ExampleBackendApplication.kt:39]: Docker compose container is example backend, which is part of how to run the application. 
Not sure what is causing the connection reset by peer, but one reproducable way is to rebuild and restart the web example. Or the example backend. I.e. not restart the full docker compose. 

So my guess it is something related to how docker compose sets up the network..

File: example-app/App.tsx [hunk 2]
```diff
@@ -53,30 +118,51 @@ const App = () => {
...
-    if (error) {
-      return {success: false, errorReason: error};
+    console.log('url', deeplinkUrl);
+
+    let path = deeplinkUrl.pathname;
+    if( deeplinkUrl.protocol !== "http:" ){
+      // custom protocol like truidtest://
+      // then the first part of pathname is parsed as host
+      path = `/${deeplinkUrl.hostname}${path}`
     }
```
[the remark this answers is not in the document]
Comment (carabj) [reply example-app/App.tsx:153]: ah, ok, yes..

**Review by marten-w — APPROVED** [review]

File: example-backend/src/main/kotlin/app/truid/example/examplebackend/ExampleBackendApplication.kt [hunk 3]
```diff
@@ -4,9 +4,12 @@ import org.springframework.beans.factory.annotation.Value
 import org.springframework.boot.autoconfigure.SpringBootApplication
 import org.springframework.boot.runApplication
 import org.springframework.context.annotation.Bean
+import org.springframework.http.client.reactive.ReactorClientHttpConnector
 import org.springframework.web.reactive.function.client.WebClient
 import org.springframework.web.server.session.CookieWebSessionIdResolver
 import org.springframework.web.server.session.WebSessionIdResolver
+import reactor.netty.http.client.HttpClient
+import reactor.netty.tcp.TcpClient
```
Comment (johanblumenberg) [inline example-backend/src/main/kotlin/app/truid/example/examplebackend/ExampleBackendApplication.kt:12]: unused imports?

**Review by johanblumenberg — APPROVED** [review]

**Pull request merged**